src/http/structs.rs

```diff
@@ -128,36 +143,77 @@ pub struct EntityQueryResponse {
     pub databases: Vec<EntityDatabase>,
 }
 
-impl EntityQueryResponse {
-    pub fn to_table(&self) -> Table {
+impl FormatResponse for EntityProfile {
+    fn to_table(&self) -> Table {
+        let mut table = Table::new();
+        table.set_titles(Row::new(vec![
+            Cell::new("Display name"),
+            Cell::new("Description"),
+            Cell::new("Organization"),
+            Cell::new("Location"),
+            Cell::new("Links"),
+        ]));
+
+        table.add_row(Row::new(vec![
+            Cell::new(
+                &self
+                    .display_name
+                    .as_deref()
+                    .unwrap_or("null")
+                    .replace(',', "\\,"),
```

Comment:
Throughout this function, why do we have to escape a backslash before the comma?

Reply:
If someone's display name contains a comma and it's not escaped in the CSV format, CSV parsers will interpret the part after the comma as the next column.